src/resources/projects/website/listing/listing-table.ejs.md

```diff
@@ -84,11 +84,13 @@ return listing.utilities.outputLink(item, field, value, `listing-${field}`);
 <tr <%= listing.utilities.metadataAttrs(item) %><%= onclick(item) %>>
 <% for (const field of fields){ %>
 <td>
+```
 <%= outputValue(itemNumber, field) %>
+```{=html}
```

Comment:
This leads to extra `<p>` being added as the line is read as Markdown.

Reply:
refactor a bit by making `<td>` outside the raw block.

9c43d630bc7a34f3dcf7f7465380420b2890d030